src/components/EditorFooter/index.tsx

```diff
@@ -24,7 +24,7 @@ const EditorFooter = () => {
       boxShadow="dark-lg"
       lineHeight="34px"
       left="98px"
-      zIndex={1500}
+      zIndex={1400}
```

Comment:
Can you explain why you change z-index this part?

Reply:
**When the cart page is displaying:**
Components we want to display: NavBar, cart page, side bar
Components we do not want to display: court page, top bar, price bar
So we adjusted the z-index for some of these components to achieve stack order.

Reply:
> Can you explain why you change z-index this part?

**When the cart page is displaying:**
Components we want to display: NavBar, cart page, side bar
Components we do not want to display: court page, top bar, price bar
So we adjusted the z-index for some of these components to achieve stack order.

Reply:
If you have noticed. The shadow of the footer is removed by this z-index change. Do you think is there other ways by changing the shopping cart component without removing the shadow of the footer?

<img width="900" alt="image" src="https://user-images.githubusercontent.com/12379895/192081989-a58e6aa9-20cd-49f5-aafa-ab19ce50b2e4.png">

Reply:
> If you have noticed. The shadow of the footer is removed by this z-index change. Do you think is there other ways by changing the shopping cart component without removing the shadow of the footer?
> 
> <img alt="image" width="900" src="https://user-images.githubusercontent.com/12379895/192081989-a58e6aa9-20cd-49f5-aafa-ab19ce50b2e4.png">

Sorry, we did not notice the shadow of the footer. Now this problem has been fixed.